common.js

```diff
@@ -43,6 +43,25 @@ function parseCLIArgs (argv) {
     args.out = null
   }
 
+  if (args.prebuiltAsar) {
+    if (args.asar) {
+      warning('--prebuiltAsar and --asar options are incompatible. Ignoring --asar')
+      delete args.asar
+    }
+    console.log('\n\n%s\n\n', JSON.stringify(args, null, 2))
+    if (!args.prune) {
+      warning('--prebuiltAsar and --no-prune are incompatible. Ignoring --no-prune')
+    }
+
+    if (args.ignore) {
+      warning('--prebuiltAsar and --ignore are incompatible. Ignoring --ignore')
+    }
+
+    if (!args['deref-symlinks']) {
+      warning('--prebuiltAsar and --no-deref-symlinks are incompatible. Ignoring --no-deref-symlinks')
+    }
+  }
+
```

Comment:
This should be outside of the CLI argument parsing, as I mentioned yesterday, because it will not give warnings to consumers of the JavaScript API.